### DISCUSSION TITLE
Problems implementing Winter Translate

### DISCUSSION THREAD

**Discussion by GoergGeorg** [wintercms/winter#336] [Q&A] [answered]
I have implemented without problems a web using a Bootstrap 5 template which I adapted to Winter CMS.
https://digpatho.com/
It has a language switcher which now (in local developing) I'm changing with Locale Picker component inserted in navigation.htm Partial, where I define the menu. I need to add english language besides spanish.
The main problem is that  language switcher although is showing ok, is not working.
<img width="899" alt="Captura de Pantalla 2021-10-19 a la(s) 19 38 47" src="https://user-images.githubusercontent.com/81718196/138000732-133eb83d-0940-40a8-9c52-bed73cc12024.png">
Do I need to add any kind of jquery or  ajax for it to work?
If I add "/en" or "/es" the translations take place.
I tried with "Force Url schema" without success.

Any help will be appreciated.

**Comment by LukeTowers** [comment, marked as answer]
Did you include the AJAX framework in your theme? https://wintercms.com/docs/ajax/introduction#framework-script

**Comment by robballantyne** [comment]
You can use the component tag anywhere, but you need to have the component attached to either the layout or the page (not the partial) for it to be available to handle Ajax requests.

**Reply by GoergGeorg** [reply]
Thank you both Luke and robballantyne, menu selector is working now. I had to include navigation.htm partial code  and framework component to layout.
